src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -1618,8 +1618,18 @@ ExtractRangeTableEntryWalker(Node *node, List **rangeTableList)
 List *
 pull_var_clause_default(Node *node)
 {
+#if (PG_VERSION_NUM >= 90600)
+
+	/*
+	 * PVC_REJECT_PLACEHOLDERS is now implicit if PVC_INCLUDE_PLACEHOLDERS
+	 * isn't specified.
+	 */
+	List *columnList = pull_var_clause(node, PVC_RECURSE_AGGREGATES);
+#else
 	List *columnList = pull_var_clause(node, PVC_RECURSE_AGGREGATES,
 									   PVC_REJECT_PLACEHOLDERS);
```

Comment:
This commit was copied from your branch, and it preserves existing behavior, so I thought so? I didn't give it thought other than ensuring existing behavior was preserved.